selfdrive/controls/controlsd.py

```diff
@@ -731,7 +731,7 @@ def publish_logs(self, CS, start_time, CC, lac_log):
 
     if not self.read_only and self.initialized:
       # send car controls over can
-      self.last_actuators, can_sends = self.CI.apply(CC)
+      self.last_actuators, can_sends = self.CI.apply(CC, int(sec_since_boot() * 1e9))
```

Comment:
we can also make a messaging helper function to get log mono time, copied what it does now